dismob/log.py

```
@@ -143,32 +148,25 @@ async def safe_send_message(channel: discord.TextChannel, content: str = None, e
-                interaction.followup.send(content, embed=embed, ephemeral=ephemeral),
-                route='POST /webhooks/{application_id}/{interaction_token}',
-                major_params={'application_id': interaction.application_id}
-            )
-        except Exception as e:
-            error(f"Error in the followup: {e}")
+        await safe_followup(interaction, content, embed, view, file, ephemeral)
     except Exception as e:
         error(f"Error when responding to the interaction: {e}")
 
-async def safe_followup(interaction: discord.Interaction, content: str = None, embed: discord.Embed = None, ephemeral: bool = False):
+async def safe_followup(interaction: discord.Interaction, content: str = None, embed: discord.Embed = None, view: discord.ui.View = None, file: discord.File = None, ephemeral: bool = False):
```

Ruff (0.14.1)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)